specification/app/resource-manager/Microsoft.App/stable/2025-07-01/examples/ContainerApps_Kind_CreateOrUpdate.json

```diff
@@ -0,0 +1,200 @@
+      "kind": "workflowapp",
+      "properties": {
+        "managedEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/testmanagedenv3",
+        "configuration": {
+          "activeRevisionsMode": "Single",
+          "ingress": {
+            "external": true,
+            "targetPort": 80,
+            "allowInsecure": true
+          }
+        },
+        "template": {
+          "containers": [
+            {
+              "image": "default/logicapps-base:latest",
+              "name": "logicapps-container",
+              "resources": {
+                "cpu": 1.0,
+                "memory": "2.0Gi"
+              }
+            }
+          ],
+          "scale": {
+            "minReplicas": 1,
+            "maxReplicas": 30,
+            "cooldownPeriod": 350,
+            "pollingInterval": 35
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/containerapps/testcontainerAppKind",
+        "name": "testcontainerAppKind",
+        "type": "Microsoft.App/containerApps",
+        "location": "East US",
+        "managedBy": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.Web/sites/testcontainerAppKind",
+        "kind": "workflowapp",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "runningStatus": "Running",
+          "managedEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/testmanagedenv3",
+          "environmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/testmanagedenv3",
+          "workloadProfileName": null,
+          "latestRevisionName": "testcontainerAppKind--2rltv14",
+          "latestReadyRevisionName": "testcontainerAppKind--2rltv14",
+          "latestRevisionFqdn": "testcontainerAppKind--2rltv14.nicefield-53acf186.eastus.azurecontainerapps-test.io",
+          "configuration": {
+            "activeRevisionsMode": "Single",
+            "ingress": {
+              "fqdn": "testcontainerAppKind.nicefield-53acf186.eastus.azurecontainerapps-test.io",
+              "external": true,
+              "targetPort": 80,
+              "exposedPort": 0,
+              "transport": "auto",
+              "traffic": [
+                {
+                  "weight": 100,
+                  "latestRevision": true
+                }
+              ],
+              "customDomains": null,
+              "allowInsecure": true,
+              "ipSecurityRestrictions": null,
+              "corsPolicy": null,
+              "clientCertificateMode": null,
```

Comment:
>      "clientCertificateMode": null, [](http://example.com/codeflow?start=8&length=36)

clientCertificateMode being a non nullable enum value, it should not be null, it should be an actual value, or unspecified.
If you have a default behavior, consider returning the value which best explains the default behavior